ena-submission/src/ena_deposition/create_project.py

```diff
@@ -184,7 +184,9 @@ def submission_table_start(db_config: SimpleConnectionPool, config: Config):
         db_config, table_name="submission_table", conditions=conditions
     )
     logger.debug(
-        f"Found {len(ready_to_submit)} entries in submission_table in status READY_TO_SUBMIT"
+        f"Found {len(ready_to_submit)} entries in submission_table in status READY_TO_SUBMIT: {
```

Comment:
ready_to_submit is a list of dictionaries - this will be a very long and not really readable message

Reply:
Can we just delete this?

Reply:
done

Reply:
sorry this wont work, my sql library returns all the rows in the db where this statement is valid - each row is returned as a dictionary and each row is an entry in a a list. So you would have to do sth like ", ".join([row["accession"] for row in entries_to_submit][:5])

Reply:
Hmm, `ready_to_submit` _is_ a dict no? Or are the type annotations wrong?

<img width="1369" alt="image" src="https://github.com/user-attachments/assets/8018fcb3-8378-42c8-a794-f610a2b9c6fe" />